homeassistant/components/media_source/__init__.py

```diff
@@ -145,6 +148,29 @@ async def async_browse_media(
     return item
 
 
+@bind_hass
+async def async_search_media(
```

Comment:
No need to add this to media source for now. I think we can start doing it just for media players.